dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -373,7 +372,7 @@ public void setDbUpdateDate(Timestamp dbUpdateDate) {
 
     @Override
     public int hashCode() {
-        return Objects.hash(id, lastModified, reference, hidden, valid, name, commitID);
+        return Objects.hash(id, reference, hidden, valid, name, commitID);
```

Comment:
Why do you not use last_build in here? I think you should.

Reply:
I think this has to be done in the overriding method in Tag

Reply:
Yeah, lastModified no longer exists for Version. It only exists now for WorkflowVersion.

Reply:
It looks like this will never get invoked, because both subclasses of this class override this method. I would either:

1. Get rid of this method here, or
1. Use this method in the subclasses, e.g., in Tag.java the hashCode implementation, do something like `return  Objects.hash(super.hashCode(), lastBuilt)`.

Reply:
(I prefer the second)

Reply:
Adding lastbuilt or lastmodified to the overriding method hashCode in Tag/WorkflowVersion breaks it for some reason. I tried figuring it out for a while, but since it those fields weren't being used previously, I decided to continue to not include them